src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+function selectTab(selectedTab, currentTab, onTabSelected) {
+  // Function for onClick event
+  return () => {
+    if (selectedTab.id !== currentTab.id) {
+      onTabSelected(currentTab);
+    }
+  };
+}
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((currentTab, i) => (
```

Comment:
```suggestion
          {tabs.map((currentTab) => (
```
Why do you need index? You don't use it